package/MDAnalysis/analysis/rms.py

```diff
@@ -576,22 +577,26 @@ def run(self, start=None, stop=None, step=None, progout=10, quiet=False):
         progout : int (optional)
             number of frames to iterate through between updates to progress
             output; ``None`` for no updates [10]
-        quiet : bool (optional)
-            if ``True``, suppress all output (implies *progout* = ``None``)
-            [``False``]
+        verbose : bool (optional)
+            if ``False``, suppress all output (implies *progout* = ``None``)
+            [``True``]
```

Comment:
please remove the default value here. It is already shown in the function signature.